tests/contrib/gp/test_models.py

```diff
@@ -330,9 +330,10 @@ def _pre_test_mean_function():
     def f(x):
         return 2 * x + 3 + 5 * torch.sin(7 * x)
 
-    X = torch.arange(100)
+    tensor_holder = torch.tensor([])
```

Comment:
Can you explain the change in this file? I don't understand why it's needed.

Reply:
I used pytorch master, and `torch.arange` returns long tensor instead of float as in pytorch 0.4.